src/vis/vis.js

```diff
@@ -262,29 +268,46 @@ var VisModel = Backbone.Model.extend({
     }
   },
 
+  init: function (options) {
```

Comment:
It's only called once. https://github.com/CartoDB/cartodb.js/blob/a82b9b3e227099e41e454e97d677bf2a1a478cd9/src/vis/vis.js#L276-L285